### ISSUE TITLE
url in sass

### ISSUE THREAD

**Issue by saidworks** [saidworks/C2N2#3]
I had problem while using sass it is related to url as when I set an url inside any property it does not convert well in css example:
.header
         background-image: url("images/xy.svg") 
in CSS I find:
.header {
 background-image: url("imagesxy.svg") 

any ideas?

**Comment by ejjraifihamza** [comment]
yep that is a problem, i highly recommend you to check this link out it will help alot : [https://css-tricks.com/sass-techniques-from-the-trenches/](url)

**Comment by saidworks** [comment]
Thanks for the link is useful as reference to learn sass , but could not find details on how to solve this issue...
